spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/sql/TestIcebergVersionFunction.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.spark.sql;
+
+import org.apache.iceberg.IcebergBuild;
+import org.apache.iceberg.spark.SparkCatalogConfig;
+import org.apache.iceberg.spark.SparkTestBaseWithCatalog;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class TestIcebergVersionFunction extends SparkTestBaseWithCatalog {
+
+  @Parameterized.Parameters(name = "catalogConfig = {0}")
+  public static Object[][] parameters() {
+    return new Object[][] {{SparkCatalogConfig.SPARK}};
+  }
+
+  private final String fullyQualifiedSystemNamespace;
+
+  public TestIcebergVersionFunction(SparkCatalogConfig catalogConfig) {
+    super(catalogConfig);
+    this.fullyQualifiedSystemNamespace = catalogConfig.catalogName() + ".system";
+  }
+
+  @Test
+  public void testIcebergVersionFunctionReturnsCorrectResult() {
```

Comment:
Since the `TestFunctionCatalog` suite needs a test that exercises load and everything through SQL, I think you should just move this test case there and remove this suite. It isn't doing much anyway.

Reply:
Sure thing.